pyuvdata/utils.py

```diff
@@ -791,24 +791,25 @@ def antnums_to_baseline(ant1, ant2, Nants_telescope, attempt256=False):
         number of antennas
     attempt256 : bool
         Option to try to use the older 256 standard used in
-        many uvfits files (will use 2048 standard if there are more
-        than 256 antennas). Default is False.
+        many uvfits files. If there are more than 256 antennas, the 2048
+        standard will be used unless there are more than 2048 antennas.
+        In that case, the 2147483648 standard will be used. Default is False.
```

Comment:
I think this is actually triggered by antenna numbers higher than 255/2047, not by more than 256/2048 antennas.